Publish Monthly Updates for May (#83)
rubygems/rubygems.github.io: _posts/2021-06-09-may-rubygems-updates.md

In addition to that, RubyGems saw several bug fixes and updates this month, some of which include the following:

- fixed a resolution issue where gems were being unintentionally removed from the lockfile - [#4580](https://github.com/rubygems/rubygems/pull/4580).
- shipped a fix in RubyGems to improve the reproducibility of building packages - [#4610](https://github.com/rubygems/rubygems/pull/4610).
- shipped other minor improvements, and some internal changes to our development environment like moving away from `minitest` in favor of `test-unit`.

Checkout [RubyGems](https://github.com/rubygems/rubygems/blob/master/CHANGELOG.md#3218--2021-05-25) and [Bundler](https://github.com/rubygems/rubygems/blob/master/bundler/CHANGELOG.md#2218-may-25-2021) for the full changelog of the new versions shipped this month! 

In May, Rubygems gained [132 new commits](https://github.com/rubygems/rubygems/compare/master@%7B2021-05-01%7D...master@%7B2021-05-31%7D), contributed by 10 authors. There were 2419 additions and 2118 deletions across 228 files.

## RubyGems.org News

In May, RubyGems.org saw several bug fixes and updates, some of which include the following:

> - investigated and fixed cache poisoning by using `x-forwarded-scheme` header gotten from a [HackerOne Report](https://hackerone.com/bugs?subject=rubygems&report_id=1184815).

Reviewer: ```suggestion
- investigated and fixed cache poisoning by using `x-forwarded-scheme` header. The issue was reported on HackerOne.
```
perhaps we don't need to add the link since it is private.

> - set form-action and frame-ancestor CSP policy - [#2718](https://github.com/rubygems/rubygems.org/pull/2718).

Reviewer: ```suggestion
- set form-action and frame-ancestor CSP policy to mitigate bypass of X-Frame-Options using a proxy  - [#2718](https://github.com/rubygems/rubygems.org/pull/2718).
```

- researched verified publisher implementation for package manager - [#2698](https://github.com/rubygems/rubygems.org/pull/2698#issuecomment-846356370).

> - added copy link to recovery code page and disabled continue link - [#2717](https://github.com/rubygems/rubygems.org/pull/2717).

Reviewer: - test upgrade of Elasticsearch 7 on staging and estimate downtime requirements
